docs-chef-io/content/server/ctl_chef_server.md

```diff
@@ -955,6 +955,17 @@ This subcommand has the following options:
     option only works when run on a standalone Chef Infra Server or on a
     primary backend Chef server within a legacy tier.
 
+{{< note >}}
+If the search does not work when data is present even after reindex, please verify the configurations of the search index. The following command will get the configuration of search index :
+  ```bash
```

Comment:
Add a blank line before and after each code block.

Reply:
have made the changes. Thank you.